README.md

```diff
@@ -20,3 +20,30 @@ pip install flake8-force
 
 * Specify the option via the command line: `flake8 --force-check ...`.
 * Add `force-check = True` to the flake8 configuration file.
+
+## Tips for checking Cython code
+
+While this extension "forces" flake8 to ignore problems with parsing Cython syntax as Python code, flake8 must be separately configured to permit Cython syntax through ignoring certain rules.
+The configuration below is suggested for that purpose.
+Some projects may not need to ignore every rule, depending on the use of Cython.
+```ini
+[flake8]
+filename = *.py,*.pyx,*.pxd,*.pxi
+ignore = E203,E225,E226,E227,E402,E741,E901,E999,W503,W504
+force-check = True
+```
```

Comment:
How about adding a link to https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes here so that users can see what checks are listed as ignored?